### ISSUE TITLE
setup periodic image push/rebuild

### ISSUE THREAD

**Issue by Luap99** [containers/podman-machine-os#55]
With automatic push working on main the task currently only runs post merge as part of the normal cirrus testing flow. Given there are not many PRs here but the base image may get updates we should have some cron based runs to periodically rebuild/push a new image.

The question is is how often should that run? Once day or week, something else?
Should we always push the new image even if nothing changed, if we always push a new image then the local "cache" by podman machine is not as useful as users would have to repull the image every day/week?

**Comment by baude** [comment]
If we can coordinate this with the podman COPR, I would like to see this done nightly.

**Comment by Luap99** [comment]
Yeah for main nightly makes sense as we will basically always have changes due the podman-next copr you are right.

But I guess my question is more for release branches as well? I don't think spinning this daily there would add value as it would invalidate the download cache for users and unless the base image changed there are likely no actual changes in the image.

**Comment by baude** [comment]
Agree, main only.  I have no visions of changing that.

**Comment by Luap99** [comment]
nightly builds on main are setup now via cirrus cron job
